### ISSUE TITLE
[lit] Interrupting the test suite results in a TypeError

### ISSUE THREAD

**Issue by JDevlieghere** [llvm/llvm-project#55176]
```
^C  interrupted by user, skipping remaining tests
Traceback (most recent call last):
  File "/Users/jonas/llvm/build-ra/./bin/llvm-lit", line 46, in <module>
    main(builtin_parameters)
  File "/Users/jonas/llvm/llvm-project/llvm/utils/lit/lit/main.py", line 111, in main
    selected_tests, discovered_tests = GoogleTest.post_process_shard_results(
  File "/Users/jonas/llvm/llvm-project/llvm/utils/lit/lit/formats/googletest.py", line 265, in post_process_shard_results
    start_time = start_time + elapsed_time
TypeError: unsupported operand type(s) for +: 'NoneType' and 'float'
```

**Comment by yuanfang-chen** [comment]
Thanks for the report. I'll take a look.

**Comment by yuanfang-chen** [comment]
Should be fixed by 0e554ebf029f8.

**Comment by JDevlieghere** [comment]
Thank you!
